.changeset/cool-vans-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+fix(assets): removed sepolia assets and fixed some incorrect decimals config
```

Comment:
I've adjusted the PR title to be shorted (less than 72 chars) so that CI passes
```suggestion
fix: removed sepolia assets and adjust decimal config
```